### PR TITLE
Bug fix eb

### LINKED ISSUE #112: Add delete wedding functionality
- [ ] add delete wedding endpoint
- [ ] Add delete button to wedding details page

### PR DESCRIPTION
**Pull request by ericberglund117**
# Pull Request

## What's this PR do? 

- [x] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [ ] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Fix no weddings display error
- fix missing key props error in multiple components
- allow the user to delete a wedding

## What are the relevant tickets? 
* Addresses
* Closes #112

### CODE REVIEW

File: src/apiCalls.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const getWeddingInfo = (weddingId:number) => {
 }
 
 export const getWeddings = () => {
-  return fetch('http://localhost:8000/api/v1/weddings/weddings/')
+  return fetch("http://matrimania-backend.herokuapp.com/api/v1/weddings/weddings/")
```
Comment (Kristiannmiller) [inline src/apiCalls.tsx:8]: We have a deployed back end!!! Nice work!

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -32,13 +32,14 @@ const App = () => {
   useEffect(() => {
     const allWeddings = async () => {
       const result = await getWeddings()
-      if(result.length > 0) {
+      console.log(result)
```
Comment (Kristiannmiller) [inline src/components/App/App.tsx:35]: This one was tricky. I always forget that strings have length too. Thanks for fixing that up

File: src/components/App/App.tsx [hunk 3]
```diff
@@ -50,6 +51,16 @@ const App = () => {
     allWeddings()
   }, [])
 
+  const deleteSingleWedding = async (weddingId: number) => {
```
Comment (Kristiannmiller) [inline src/components/App/App.tsx:54]: So glad we can finally delete weddings!

File: src/components/WeddingDetails/WeddingDetails.tsx [hunk 4]
```diff
@@ -1,16 +1,18 @@
 import './WeddingDetails.css';
-import React, { useState, useEffect } from 'react';
-import { getSingleWeddingGuests, getSingleWeddingPhotos, getWeddings, postAGuest } from '../../apiCalls';
+import React, { useState, useEffect, useMemo } from 'react';
```
Comment (Kristiannmiller) [inline src/components/WeddingDetails/WeddingDetails.tsx:2]: Did we get rid of this useMemo? Just being a human linter...

**Review by Kristiannmiller — APPROVED** [review]
This looks great! Thanks for adding the delete wedding functionality. We definitely needed that.

File: src/apiCalls.tsx (hunk 1 above)
Comment (hannakim91) [reply src/apiCalls.tsx:8]: Thank you @IamNorma and @fieldstyler!!

**Review by hannakim91 — APPROVED** [review]
Thanks for updating the URLs to the new deployed API and writing out the new delete weddings feature!

**Pull request merged**